src/Features/Core/Portable/GenerateConstructors/AbstractGenerateConstructorsCodeRefactoringProvider.ConstructorDelegatingCodeAction.cs

```diff
@@ -9,21 +9,24 @@
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeGeneration;
 using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.GenerateFromMembers;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.GenerateConstructorFromMembers;
+namespace Microsoft.CodeAnalysis.GenerateConstructors;
 
-internal abstract partial class AbstractGenerateConstructorFromMembersCodeRefactoringProvider
+using static GenerateFromMembersHelpers;
+
+internal abstract partial class AbstractGenerateConstructorsCodeRefactoringProvider
```

Comment:
renamed this type to indicate it's for any sort of 'generate constructor' refactoring expereince.  not just generating based on members.